<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterformationwindow</class>
 <widget class="QDialog" name="ajouterformationwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblCodeFormation">
        <property name="text">
         <string>Code de la formation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtCodeFormation">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblDescription_Formation">
        <property name="text">
         <string>Titre</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="txtDescriptionFormation">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblCreditsFormation">
        <property name="text">
         <string>Crédits requis</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTableWidget" name="tableFormationCreditsNeeded">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Catégorie</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nombre de crédits</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblCreditsFormation_2">
        <property name="text">
         <string>Formations requises</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTableWidget" name="tableFormationFormationsNeeded">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Formation</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Prérequis ?</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblCreditsFormation_3">
        <property name="text">
         <string>UV requises</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTableWidget" name="tableFormationUVNeeded">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>UV</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Prérequis ?</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblCreditsFormation_4">
        <property name="text">
         <string>Crédits requis dans liste d'UV</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QTableWidget" name="tableFormationCreditsNeededInUVSet_Credits">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Catégorie</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nombre de crédits</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QTableWidget" name="tableFormationCreditsNeededInUVSet_UV">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>405</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>UV</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Dans la liste ?</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnAccepter">
        <property name="text">
         <string>Ajouter la nouvelle formation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAnnuler">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
